common/changes/@microsoft/rush/rebuild_2021-07-09-20-31.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "@microsoft/rush",
+      "comment": "rush rebuild ignores existing build cache entries",
+      "type": "none"
+    }
```

Comment:
Let's also add a second change log that is like:
```
"(Breaking change) Remove the experimental \"--disable-build-cache\" command line parameter."
```

Although the feature is still experimental, these are nontrivial changes, so we should remember to publish them as a MINOR bump.